client/post-editor/editor-seo/accordion.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import CountedTextarea from 'components/forms/counted-textarea';
+import Gridicon from 'components/gridicon';
+import InfoPopover from 'components/info-popover';
+import TokenField from 'components/token-field';
+import { getSiteSlug } from 'state/sites/selectors';
+
+class SeoAccordion extends Component {
+	constructor() {
+		super();
+
+		this.displayName = 'EditorSeoAccordion';
+	}
+
+	render() {
+		const { translate } = this.props;
+		// Temporary placeholder chips for design review
+		const sampleChips = [ 'Post Title', 'Site Title' ];
+
+		return (
+			<Accordion
+				title={ translate( 'Advanced SEO' ) }
+				icon={ <Gridicon icon="search" /> }
+				className="editor-drawer__editor-seo"
```

Comment:
Still not quite right. Per the CSS recommendations, the component fragment should always match the directory name, which is `editor-seo`.

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines